.werft/build.ts

```diff
@@ -351,6 +352,9 @@ export async function build(context, version) {
         werft.done(vmSlices.KUBECONFIG)
 
         exec(`kubectl apply -f clouddns-dns01-solver-svc-acct.yaml -f letsencrypt-issuer.yaml`, { slice: vmSlices.INSTALL_LETS_ENCRYPT_ISSUER, dontCheckRc: true })
+
+        VM.installFluentBit({namespace: 'default', slice: vmSlices.EXTERNAL_LOGGING});
+
```

Comment:
```suggestion
        VM.installFluentBit({namespace: 'default', slice: vmSlices.EXTERNAL_LOGGING});
        werft.done(vmSlices.EXTERNAL_LOGGING)
```

It's important to close slices so we get nice traces in honeycomb :) 
See also: https://github.com/gitpod-io/gitpod/pull/7984